content/es/ebpf.md

```diff
@@ -0,0 +1,41 @@
+---
+título: eBPF
+estado: Completado
+categoría: Tecnología
+etiquetas: ["arquitectura", "redes", "seguridad"]
```

Comment:
It's tricky, I know but you can see the section between the `---` like something that both human and machine "reads". That's why everything on the side of the `:` has to stay the same, since the software will look for those specific keywords, while the right size can be translated (with some exceptions like `status`).

Reply:
My understanding is because there is a [page](https://glossary.cncf.io/es/tags/) where you can search concepts by category, so that's why those terms have to be also translated but keeping the keys as they are in English